ember.md

```diff
@@ -85,11 +87,37 @@ get(this, 'isDestructured'); //true
 set(someObject, 'isUpdated', true);
 get(someObject, 'isUpdated'); //true
 ```
+
+<a name="computed-properties"></a>
+## Computed Properties
+
+<a name="computed-properties--brace-expansion"></a>
+### 3.1 Brace Expansion
+
+When a computed property depends on multiple properties, we can declare the dependents using brace expansion. It is useful for cases when the computed property depends on multiple properties of an object. Using brace expansion in computed properties makes our code more organized and easier to read, as it organizes dependent keys. 
+
+```javascript 
+// Bad 
+fullname: computed('user.firstname', 'user.lastname', function() {
+  const firstname = get(this, 'user.firstname'); 
```

Comment:
Taking @ASH-Anthony's lead on the suggestion to keep the template strings in the return statement of the "bad" example, I would also use the destructured const declaration (`const { firstname, lastname } = get(this, 'user');`) in the bad example since this section is to point out the use of brace expansion.